geowebcache/core/src/main/java/org/geowebcache/mime/MimeType.java

```diff
@@ -198,7 +198,8 @@ public int hashCode() {
 
 
     public boolean isCompatible(String otherMimeType) {
```

Comment:
Could you update the javadoc to clarify how this is used.  Something like "Checks if the given MIME type is a subtype of this one."